app/views/pages/aboutus.html.erb

```diff
@@ -1,4 +1,4 @@
-<strong><h2>About Us</h2></strong>
+<strong><h2><%= @page.name 'About Us' %></h2></strong>
```

Comment:
I think we need more investigation here before we can merge this - seems like strange duplication